lib/kcl.js

```diff
@@ -231,7 +230,8 @@ module.exports = function(config, kinesis) {
         if (!resp.data.NextShardIterator && resp.data.Records) {
           db.shardComplete(shard, function (err) {
             if (err) throw err;
-            console.log('Shard', shard.id, 'marked as complete');
+            shard.status = 'complete';
```

Comment:
should we send the event before marking the shard as complete in the database?  That way if something fails in the host application's onShardClosed function it will get a second chance when the application restarts